src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV36.cs

```diff
@@ -0,0 +1,148 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using FellowOakDicom;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.Core.Models;
+using Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Storage;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.Store;
+
+/// <summary>
+/// Sql IndexDataStore version 36.
+/// </summary>
+internal class SqlIndexDataStoreV36 : SqlIndexDataStoreV35
+{
+    public SqlIndexDataStoreV36(SqlConnectionWrapperFactory sqlConnectionWrapperFactory)
+        : base(sqlConnectionWrapperFactory)
+    {
+    }
+
+    public override SchemaVersion Version => SchemaVersion.V36;
+
+    public override async Task EndCreateInstanceIndexAsync(
```

Comment:
I got anxious because I updated these sprocs in two ways:
1. Moved the `ChangeFeed` `INSERT` to the end of the transaction
2. Use the new default column constraint to generate the timestamps upon `INSERT` instead of relying on one computed at the start of the transaction

(2) made me nervous about whether there was some race condition between the sproc being updated but the table not having the constraint yet... but I don't need to worry given the order of operations in the migration script